src/components/application_manager/src/commands/hmi/get_urls.cc

```diff
@@ -93,7 +93,6 @@ void GetUrls::ProcessPolicyServiceURLs(const policy::EndpointUrls& endpoints) {
 
   if (!app_id_to_send_to) {
     SDL_ERROR("There are no available applications for processing.");
```

Comment:
@orhan-mehmedov 
SDL must respond to HMI with some error, so in that case you don't have to completely remove that, but use SendResponseToHMI() method instead. As I understood problem is that with plain ManageHMICommand type of message stays the same - kRequest, so that causes loop.